support/ruby/consolr/spec/consolr_spec.rb

```diff
@@ -0,0 +1,133 @@
+      end
+    end
+
+    it 'does log list' do
+      expect { @console.start({:tag => safe_maintenance_tag, :log => 'list'}) }.to output("sel list\nSUCCESS\n").to_stdout_from_any_process
+    end
+
+    it 'does log clear' do
+      expect { @console.start({:tag => safe_maintenance_tag, :log => 'clear'}) }.to output("sel clear\nSUCCESS\n").to_stdout_from_any_process
+    end
+  end
+
+  describe 'dangerous allocated asset' do
+    dangerous_allocated_tag= 'dangerous-allocated-tag'
+    safe_boolean_actions.each do |action, response|
+      it "does #{action}" do
+        expect { @console.start({:tag => dangerous_allocated_tag, action => true}) }.to output("#{response}\nSUCCESS\n").to_stdout_from_any_process
+      end
+    end
+
+    dangerous_boolean_actions.each do |action, response|
+      it "refuses to do dangerous action #{action}" do
+        expect { @console.start({:tag => dangerous_allocated_tag, action => true}) }.to raise_error(SystemExit, /Asset .* is a crucial asset/)
+      end
+      it "refuses to do dangerous action #{action} with force" do
+        expect { @console.start({:tag => dangerous_allocated_tag, action => true, :force => true}) }.to raise_error(SystemExit, /Asset .* is a crucial asset/)
+      end
+    end
+
+    it 'does log list' do
+      expect { @console.start({:tag => dangerous_allocated_tag, :log => 'list'}) }.to output("sel list\nSUCCESS\n").to_stdout_from_any_process
+    end
+
+    it 'does log clear' do
+      expect { @console.start({:tag => dangerous_allocated_tag, :log => 'clear'}) }.to output("sel clear\nSUCCESS\n").to_stdout_from_any_process
+    end
+  end
+
+  describe 'dangerous maintenance asset' do
+    dangerous_maintenance_tag= 'dangerous-maintenance-tag'
+    safe_boolean_actions.each do |action, response|
+      it "does #{action}" do
+        expect { @console.start({:tag => dangerous_maintenance_tag, action => true}) }.to output("#{response}\nSUCCESS\n").to_stdout_from_any_process
+      end
+    end
+
+    dangerous_boolean_actions.each do |action, response|
+      it "refuses to do dangerous action #{action}" do
+        expect { @console.start({:tag => dangerous_maintenance_tag, action => true}) }.to raise_error(SystemExit, /Asset .* is a crucial asset/)
+      end
+      it "refuses to do dangerous action #{action} with force" do
+        expect { @console.start({:tag => dangerous_maintenance_tag, action => true, :force=> true}) }.to raise_error(SystemExit, /Asset .* is a crucial asset/)
+      end
+
+    end
+
+    it 'does log list' do
+      expect { @console.start({:tag => dangerous_maintenance_tag, :log => 'list'}) }.to output("sel list\nSUCCESS\n").to_stdout_from_any_process
+    end
+
+    it 'does log clear' do
+      expect { @console.start({:tag => dangerous_maintenance_tag, :log => 'clear'}) }.to output("sel clear\nSUCCESS\n").to_stdout_from_any_process
+    end
```

Comment:
`log` takes an argument (`list` or `clear`). The others take booleans.  I think we should change the way its structured, but this PR was already too big.